backend/routes/users.js

```diff
@@ -50,11 +50,27 @@ users.get('/:id', async ctx => {
 });
 
 users.patch('/:id', parsers.json, async (ctx) => {
-    try{ctx.body = await Users.query()
-        .patch({isVerified: true})
-        .findById(ctx.params.id)}
+    let response = {}
+    try{
+        let targetUser = await Users.query()
+        .findById(ctx.params.id)
+
+        //currently, we have to have dateRegistered and lastLogin in patch request or it won't work
```

Comment:
I think we shouldn't have this constraint. What error do we get if we don't have these? We could maybe tweak Objection or the DB model.

Reply:
I was messing around with that for a while and couldn't figure that out, only worked when they were both in the payload. The error code is SQL 22007.